Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -110,7 +59,7 @@ if($deployFilesInParallel -eq "false")
         $fdqn = $resource.Name
         Write-Output "Copy will be Started for - $fdqn"
```

Comment:
Write-Output and write-host both appear to the console in the build definition summary console view?